.rubocop.yml

```diff
@@ -15,6 +11,7 @@ AllCops:
   Exclude:
     - !ruby/regexp /db\/migrate\/(201|202[0-4])/
   StringLiteralsFrozenByDefault: true
+  SuggestExtensions: false
```

Comment:
Adding this to avoid the following warning:

```
The following RuboCop extension libraries are installed but not loaded in config:
  * rubocop-capybara
  * rubocop-factory_bot
  * rubocop-rspec
  * rubocop-rspec_rails

You can opt out of this message by adding the following to your config (see https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions for more options):
  AllCops:
    SuggestExtensions: false
```